molgenis-data-mapper/src/main/resources/templates/view-attribute-mapping-feedback.ftl

```diff
@@ -30,29 +30,29 @@
 	    							</#if>
 								<#else>
 									<#if feedbackRow.sourceEntity.get(sourceAttribute.name)??>
-	    								<td>${feedbackRow.sourceEntity.get(sourceAttribute.name)?html}</td>
+                                        <td>${feedbackRow.sourceEntity.get(sourceAttribute.name)?html}</td>
 	    							</#if>
 								</#if>
     						</#list>
     					</#if>
     					<#if feedbackRow.success>
     						<#if feedbackRow.value??>
-    							<#if feedbackRow.value?is_date_like>
-    								<td>${feedbackRow.value?datetime}</td>
-    							<#else>
-									<#if feedbackRow.value?is_sequence>
-										<td>
-											<#list feedbackRow.value as row>
-												${row?html}<#if row?has_next>, </#if>
-											</#list>
-										</td>
-    								<#else>
-										<td>${feedbackRow.value?html}</td>
-									</#if>
-    							</#if>
-    						<#else>	
-    							<td><i>null</i></td>
-    						</#if>
+								<#if feedbackRow.value?is_date_like> <!-- its a date or datetime -->
+									<td>${feedbackRow.value?datetime}</td>
+								<#elseif feedbackRow.value?is_hash> <!-- its a dynamic entity -->
```

Comment:
can't it be a static entity as well? "it's an xref" ;) (and yeah, I agree this is not really a big deal...)